applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -56,11 +56,11 @@ Complete the following steps to configure the module:
 
    Those options affect :ref:`nrf_desktop_usb_state` that subscribes for HID boot reports.
    The Dongle forwards HID reports from both mouse and keyboard, and so either option works if you want to have the Dongle work as boot mouse or boot keyboard.
-   For more information about the configuration of the HID boot protocol, see the boot protocol configuration section in the :ref:`nrf_desktop_usb_state` documentation.
+   For more information about the configuration of the HID boot protocol, see the Boot protocol configuration section in the :ref:`nrf_desktop_usb_state` documentation.
```

Comment:
```suggestion
   For more information about the configuration of the HID boot protocol, see the `Boot protocol configuration` section in the :ref:`nrf_desktop_usb_state` documentation.
```

Reply:
Single backtics do nothing. Quotation marks maybe or nothing.

Reply:
Quotation marks should be ok too. Just to separate the title visually.